ui/src/main/webapp/src/app/analysis-context/analysis-context-form.component.ts

```diff
@@ -62,7 +62,7 @@ export class AnalysisContextFormComponent extends FormComponent
         },
         {
             "id": 100,
-            "name": "Migration to Red Hat JBoss EAP 6",
+            "name": "Migration to JBoss EAP 6",
             "source": null,
```

Comment:
There is missing change https://docs.google.com/spreadsheets/d/1Xo1v_VpZmCfLIx5ySX-gyBK1w0uPTdntHofqRNU5fNE/edit#gid=0&range=E19
None -> Cloud readiness only ? https://github.com/klinki/windup-web/blob/5d79e999a638f97f7abae9bdab0d2bebb59df4cb/ui/src/main/webapp/src/app/analysis-context/analysis-context-form.component.ts#L76